docs/content/usage/scenarios/_index.md

```diff
@@ -0,0 +1,29 @@
++++
+title = "Scenarios"
+date = 2020-08-12T12:52:59+03:00
+weight = 3
+chapter = true
+pre = "<i class='fas fa-map-marked-alt'></i> "
++++
+
+# Scenarios
+
+This section describes the different attack <i>Scenarios</i> that the Infection Monkey can simulate.
+
+{{% notice note %}}
+Don't worry! The Infection Monkey uses safe exploiters and does not cause any permanent system modifications that could impact security or operations.
+{{% /notice %}}
+
+The Infection Monkey has pre-built scenarios to simulate common types of attacks that take place. These scenarios, when selected, manipulate the default configuration values and only show the configuration fields relevant to the selected scenario. This makes it possible for you to quickly run the Monkey on your network in order to accomplish a specific objective.
+
+![Choose scenario](/images/usage/scenarios/choose-scenario.png "Choose a scenario")
+
+To exit a scenario and select another one, click on "Start Over".
+
+![Start over](/images/usage/scenarios/start-over.png "Start over")
+
+You can also refer to [our FAQ](../../faq) for more specific questions and answers.
```

Comment:
Does FAQ contain something explicit to scenarios? If yes, then link to that specific bulltepoint. If not, no need to mention it here

Reply:
Removed